Commands.md

```diff
@@ -221,11 +221,12 @@ project  : git-extras
 
 The `--line` option can also take a path, which will print a filtered summary for that folder or file.
 
-The option `--oneline` tries to put as much summary information of the repo into a single output line
```

Comment:
I would suggest keeping the oneline output as another output style.

Reply:
Done. Now both styles are supported